webbpsf/tests/validate_webbpsf.py

```diff
@@ -99,7 +99,7 @@ def validate_vs_russ_plot7(base_opd='OPD_RevV_nircam_155.fits'):
     P.draw()
```

Comment:
This whole file validate_webbpsf is another one to delete. It's leftover testing from long ago in very early development of webbpsf. Separate PR.